src/app/Random.js

```diff
@@ -0,0 +1,41 @@
+const recipe = document.querySelector("#recipiesRandom");
+
+function getRandomRecipes() {
+    recipe.replaceChildren();
+
+    fetch('https://api.spoonacular.com/recipes/random?apiKey=fa79327224724e9da0b733fdcc9720d4')
```

Comment:
moving api key to variable seems like a good idea

Reply:
ok